src/Elastic.Apm/Api/Request.cs

```diff
@@ -22,7 +23,9 @@ public class Request
 		[Sanitize]
```

Comment:
Does this also need to be sanitized by the filter too?

Looks like this is the only usage of `SanitizeAttribute` now, so can probably be removed, since it is not used for anything (used to be used by the contract resolver to determine converter to use).

Reply:
Good point!

> Does this also need to be sanitized by the filter too?

No - body sanitization happens when we read the request body in `HttpRequestExtensions.ExtractRequestBody`. And that only happens if the body is form content type. Originally we did body sanitization during serialization, but we changed to do it during body capturing after some iterations on the problem.

Reason is that once the body is captured, we'd need to parse it again and do the serialization which is super hard (with form encoded it's easy, but if it's not form encoded we can ran into all sorts of problems), plus it's very high overhead.

> Looks like this is the only usage of SanitizeAttribute now, so can probably be removed, since it is not used for anything (used to be used by the contract resolver to determine converter to use).

I agree, I ended up just removing it. So in sum: for body sanitization nothing really changes in this PR, except that we remove this attribute, but the way we do sanitization for the body is the same as before.